### PR TITLE
[vrops-exporter] alerts prio 3 added

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/host.alerts [hunk 1]
```diff
@@ -199,28 +199,84 @@ groups:
...
+  - alert: APodMemoryUtilizationHigh
     expr: vrops_hostsystem_memory_usage_percentage{vccluster=~".*management.*"} > 90
     for: 30m
     labels:
       severity: info
       tier: vmware
       service: compute
       context: "{{ $labels.hostsystem }} memory"
-      meta: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+      meta: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }})"
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/alerts/host.alerts:210]: Is the cluster not relevant anymore?

File: prometheus-exporters/vrops-exporter/alerts/host.alerts [hunk 2]
```diff
@@ -199,28 +199,84 @@ groups:
...
+
+  - alert: HostHighNumberOfPacketsDropped
+    expr: vrops_hostsystem_alert_info{alert_name="Host is experiencing high number of packets dropped"}
+    labels:
+      severity: info
+      tier: vmware
+      service: compute
+      context: "{{ $labels.hostsystem }} packets dropped"
+      meta: "`{{ $labels.hostsystem }}` is experiencing high number of packets dropped. ({{ $labels.vcenter }})."
+      playbook: docs/devops/alert/vcenter
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/alerts/host.alerts:236]: Given alert is not yet covered, same for the others. Would rather remove the playbook to keep it in mind.

File: prometheus-exporters/vrops-exporter/alerts/host.alerts [hunk 3]
```diff
@@ -199,28 +199,84 @@ groups:
...
+      no_alert_on_absence: "true"
+    annotations:
+      description: "`{{ $labels.hostsystem }}` is experiencing high number of packets dropped. ({{ $labels.vcenter }})."
+      summary: "`{{ $labels.hostsystem }}` is experiencing high number of packets dropped. ({{ $labels.vcenter }})."
+
+  - alert: HostNicLinkFlappingAlert
+    expr: vrops_hostsystem_alert_info{alert_name=~"ESXi host has detected a link status.*"}
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/alerts/host.alerts:243]: Is the wildcard really needed? The other `alert_names` seem to match straight.

File: prometheus-exporters/vrops-exporter/alerts/nsxt.alerts [hunk 4]
```diff
@@ -211,6 +211,19 @@ groups:
...
 
+  - alert: NSXTNodeConnectivityBroken
+    expr: vrops_nsxt_mgmt_node_connectivity_status{state="DISCONNECTED"}
+    for: 5m
+    labels:
+      severity: info
+      tier: vmware
+      service: compute
+      meta: "NSX-T Node `{{ $labels.nsxt_mgmt_node }}` connectivity is broken. ({{ $labels.nsxt_adapter }})"
+      playbook: docs/devops/alert/nsxt
```
Comment (viennaa) [inline prometheus-exporters/vrops-exporter/alerts/nsxt.alerts:222]: missing too

**Review by viennaa — CHANGES_REQUESTED** [review]
Lots of alerts floating around 💧

File: prometheus-exporters/vrops-exporter/alerts/host.alerts (hunk 1 above)
Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/host.alerts:210]: I would like to keep it as clean as possible. The cluster is also known throughout the `hostsystem` label.

File: prometheus-exporters/vrops-exporter/alerts/host.alerts (hunk 2 above)
Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/host.alerts:236]: Sure. Thanks for the hint.

File: prometheus-exporters/vrops-exporter/alerts/host.alerts (hunk 3 above)
Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/host.alerts:243]: Yes. This alert is s bit special. The alert name is: `alert_name="ESXi host has detected a link status "flapping" on a physical NIC"` 

Double quotes.

Comment (richardtief) [reply prometheus-exporters/vrops-exporter/alerts/host.alerts:243]: Shall I do so: `alert_name="ESXi host has detected a link status /"flapping"/ on a physical NIC"` ?

Comment (viennaa) [reply prometheus-exporters/vrops-exporter/alerts/host.alerts]: Yes, put in the whole expected name please.

**Review by viennaa — APPROVED** [review]
Thanks, good to go.

**Pull request merged**